src/web/lib/components/AppHeader/index.scss

```diff
@@ -12,7 +12,7 @@
   z-index: 2;
 
   .app-header__content {
-    align-items: center;
+    justify-content: center;
```

Comment:
Nit: Is this an intentional design change? I don't want to second guess it, if so. Just looks different to me.

Reply:
Yes, @caitmuenster suggested we center the 'Firefox Color' logo & text after removing the Test Pilot text from the header.

Reply:
All good then!

Reply:
Thanks, @lmorchard ! :)